src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+	<div class="loan-search-filters">
+		<div class="filter-list">
+			<h3>Filters:</h3>
+			<ul>
+				<li @click="showFilters">
+					Gender
+				</li>
+				<!-- <li @click="showFilters">
+					Location
+				</li> -->
+				<li @click="showFilters">
+					Sector
+				</li>
+			</ul>
+		</div>
+
+		<kv-lightbox
+			class="filter-controls-lightbox"
+			id="filterControlsLightbox"
+			:visible="filtersVisible"
+			@lightbox-closed="filtersVisible = false"
+		>
+			<gender-filter
+				:initial-gender="initialGender"
+				@updated-filters="handleUpdatedFilters"
+			/>
+
+			<sector-filter
+				:initial-sectors="initialSectors"
+				@updated-filters="handleUpdatedFilters"
+			/>
+
+			<kv-button
+				class="button smallest"
+				@click.native.prevent="applyFilters"
+			>
+				Apply Filters
+			</kv-button>
+		</kv-lightbox>
+	</div>
+</template>
+
+<script>
+// import gql from 'graphql-tag';
+import KvButton from '@/components/Kv/KvButton';
+import KvLightbox from '@/components/Kv/KvLightbox';
+import GenderFilter from '@/components/CorporateCampaign/LoanSearch/GenderFilter';
+import SectorFilter from '@/components/CorporateCampaign/LoanSearch/SectorFilter';
+
+export default {
+	components: {
+		KvButton,
+		KvLightbox,
+		SectorFilter,
+		GenderFilter,
+	},
+	props: {
+		initialFilters: {
+			type: Object,
+			default: () => {}
+		},
+	},
+	data() {
+		return {
+			filtersVisible: false,
+			// gender: 'both',
+			updatedFilters: null,
+			modifiedFilters: null,
+		};
+	},
+	methods: {
+		showFilters() {
+			this.filtersVisible = true;
+		},
+		// updateFilter(filter, value) {
+		// 	if (this.updatedFilters[filter] !== value) {
+		// 		this.updatedFilters[filter] = value;
+		// 		this.$emit(
+		// 			'new-filters-selected',
+		// 			{
+		// 				filter,
+		// 				filters: this.updatedFilters
+		// 			}
+		// 		);
+		// 	}
+		// },
+		handleUpdatedFilters(payload) {
+			console.log('handleUpdatedFilters: ', payload);
+			const filterKeys = Object.keys(payload);
+			console.log('filterKeys: ', filterKeys);
+			filterKeys.forEach(key => {
+				this.modifiedFilters[key] = payload[key];
+			});
+		},
+		applyFilters() {
+			console.log(this.modifiedFilters);
+			this.$emit('updated-filters', this.modifiedFilters);
+			this.filtersVisible = false;
+		},
+		copyFilters(initialFilters) {
```

Comment:
We'll prob move this to a util file later. It makes a plain object copy of the initial reactive filters.